### PR TITLE
fox(webapp): remove decimal, remove negative number, maximum of 3 num…

### LINKED ISSUE #480: Fix textfield cost tokens offer
This should not receive a semicolon

To test, follow next steps (Under Sponsor profile):

1. Go to offer management table
2. Select add new offer
3. Check textfield toke cost is not accepting semicolon symbol

### PR DESCRIPTION
**Pull request by leisterfrancisco**
Resolves #480 

Steps to test:
1. make install
1. make run

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (JustinCast) [inline package.json:1]: Why this file is in the root level of the project?

Comment (leisterfrancisco) [reply package.json:1]: you right, I'm gonna change it

**Review by JustinCast — APPROVED** [review]

**Review by AngeloCG97 — APPROVED** [review]

**Review by steph222 — APPROVED** [review]

**Review by Edgar097 — APPROVED** [review]

**Review by tetogomez — APPROVED** [review]
LGTM

**Review by xavier506 — COMMENTED** [review]
LGTM

**Pull request merged**